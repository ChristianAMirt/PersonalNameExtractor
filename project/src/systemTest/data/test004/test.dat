<NER> Robert Smith and Maria Garcia should have per markers.</NER>