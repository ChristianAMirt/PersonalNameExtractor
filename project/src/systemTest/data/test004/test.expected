<NER> <PER> Robert Smith </PER> and <PER> Maria Garcia </PER> should have per markers .</NER> 
